omp/matrix/coo_kernels.cpp

```diff
@@ -167,6 +167,16 @@ GKO_INSTANTIATE_FOR_EACH_VALUE_AND_INDEX_TYPE(
     GKO_DECLARE_COO_CONJ_TRANSPOSE_KERNEL);
 
 
+template <typename ValueType, typename IndexType>
+void convert_to_csr(std::shared_ptr<const OmpExecutor> exec,
+                    matrix::Csr<ValueType, IndexType> *result,
+                    const matrix::Coo<ValueType, IndexType> *source)
+    GKO_NOT_IMPLEMENTED;
```

Comment:
You should be able to do the same here that you do in the reference implementation (so only `convert_row_idxs_to_ptrs`) is not implemented.